microcosm_pubsub/handlers.py

```diff
@@ -122,3 +126,22 @@ def get_resource(self, message, uri):
 
     def handle(self, message, uri, resource):
         return True
+
+
+@binding("publish_message_batch")
+@handles(created("BatchMessage"))
+@logger
+class PublishBatchMessage(object):
```

Comment:
I'm guessing we want `object` here for Python2 compatibility?

Reply:
Until we decide to stop supporting Python2.